openwisp_monitoring/device/api/views.py

```diff
@@ -246,13 +246,32 @@ class MonitoringGeoJsonLocationList(GeoJsonLocationList):
 
 monitoring_geojson_location_list = MonitoringGeoJsonLocationList.as_view()
 
+class PaginationWithFloorList(ListViewPagination):
+
+    def get_available_floors(self):
+        location_id = self.request.parser_context["kwargs"].get("pk")
+        floors = (
+            DeviceLocation.objects
+            .filter(location__id=location_id, floorplan__isnull=False)
+            .values_list("floorplan__floor", flat=True)
+            .distinct()
+            .order_by("floorplan__floor")
+        )
```

Comment:
This will bypass the filtering done by the `MonitoringLocationDeviceList` view. Is there a way to access the queryset of the view here?